markup_fmt/tests/fmt/html/text-wrapping/fixture.default.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: markup_fmt/tests/fmt.rs
+---
+<a
+  href="quite-a-long-url-that-just-keeps-going"
+>short text</a>
+<a
+  href="quite-a-long-url-that-just-keeps-going"
+>short text</a>
+
+<a
+  href="not-a-long-url"
+>but a long piece of text that wraps</a>
```

Comment:
Thanks for looking at this so quickly!
I like this change overall - it feels much more readable.

I would prefer it if the second one here would come out as

```html
<a href="not-a-long-url"
>but a long piece of text that wraps</a>
```

where only the text is broken over the line, since the attributes don't need to wrap

Reply:
Updated.